docs/html/setup/README.md

```diff
@@ -1,44 +1,43 @@
 # Setup
-Tonight we will learn front-end architecture and HTML.
+
+Today we will learn what web development is and HTML.
 
 Follow the instructions on this page to prepare your workspace.
 
->[!TIP]
->Open [Slack](http://kcwit.slack.com/) to the **#codingandcocktails** channel. It's a great way to stay in touch with your Coding & Cocktails friends and to ask questions during and after the session. We'll also post updates and tips in Slack if we run in to any stumbling blocks tonight.
+> [!TIP]
+> Open [Slack](http://kcwit.slack.com/) to the **#codingandcocktails** channel. It's a great way to stay in touch with your Coding & Cocktails friends and to ask questions during and after the session. We'll also post updates and tips in Slack if we run in to any stumbling blocks today.
 
->[!WARNING]If you are borrowing a laptop, please ask your mentor to explain the tools.
+> [!WARNING]If you are borrowing a laptop, please ask your mentor to explain the tools.
 
->[!WARNING]
->If you are using a Chromebook, having trouble installing tools, or otherwise need to use a Cloud IDE option, we will use a cloud development environment called [**CodeSandbox**](https://codesandbox.io). In order to use CodeSandbox, you will need a GitHub account. 
+> [!WARNING]
+> If you are using a Chromebook, having trouble installing tools, or otherwise need to use a Cloud IDE option, we will use a cloud development environment called [**StackBlitz**](https://stackblitz.com/). In order to use StackBlitz, you will need a GitHub account.
 >
-> Navigate to GitHub to create a personal account or log in. Feel free to ask your mentor for help! When complete and logged in to GitHub, return here to continue the instructions. You won't install any tools listed on this page, but please read through them so you have context for tonight's worksheet.
-
+> Navigate to GitHub to create a personal account or log in. Feel free to ask your mentor for help! When complete and logged in to GitHub, return here to continue the instructions. You won't install any tools listed on this page, but please read through them so you have context for today's worksheet.
 
 # Required tools for this session
 
-You don't need all the tools for tonight's session. If you haven't installed all the tools listed in [The Tools](https://bit.ly/CnCTheTools) guide already, use the following list to install only the required tools for the worksheet:
-
->[!WARNING]If you are borrowing a laptop, your tools are pre-installed. Please ask your mentor to explain the tools we are using tonight.
-
+You don't need all the tools for today's session. If you haven't installed all the tools listed in [The Tools](https://bit.ly/CnCTheTools) guide already, use the following list to install only the required tools for the worksheet:

```

Comment:
You might want to consider removing this so tools can be approached session by session. And then you'll need to bring in any tools directly into here. Up to you on how you want to handle, but if there's the big tools section, the expectation is that defines all the tools needed for the entire set of sessions.